packages/caip/src/adapters/gem/index.ts

```diff
@@ -0,0 +1,35 @@
+import toLower from 'lodash/toLower'
+
+const CAIP19ToGemAssetIdMap = {
+  'bip122:000000000019d6689c085ae165831e93/slip44:0': 'binance-coin',
+  'eip155:1/slip44:60': 'ethereum',
+  'eip155:1/erc20:0x7fc66500c84a76ad7e9c93437bfc5ac33e2ddae9': 'aave',
+  'eip155:1/erc20:0x0d8775f648430679a709e98d2b0cb6250d2887ef': 'basic-attention-token',
+  'eip155:1/erc20:0x4fabb145d64652a948d72533023f6e7a623c7c53': 'busd',
+  'eip155:1/erc20:0xc00e94cb662c3520282e6f5717214004a7f26888': 'compound-coin',
+  'eip155:1/erc20:0xd533a949740bb3306d119cc777fa900ba034cd52': 'curve-dao',
+  'eip155:1/erc20:0x6b175474e89094c44da98b954eedeac495271d0f': 'dai',
+  'erc20:0x056fd409e1d7a124bd7017459dfea2f387b6d5cd': 'gemini-dollar',
```

Comment:
ser

Reply:
Nice one, I legit re-read it 10 times before I saw the missing part of the caip here